src/app.py

```diff
@@ -16,5 +16,46 @@
 
 from celery.app import Celery
 
+from opentelemetry import trace
+
+from opentelemetry.exporter.otlp.proto.grpc import trace_exporter as grpc_exporter
+from opentelemetry.exporter.otlp.proto.http import trace_exporter as http_exporter
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+from opentelemetry.instrumentation.celery import CeleryInstrumentor
+
+def setup_telemetry(service_name: str):
```

Comment:
* Move to openrelik-common library
* Make sure to add good library comments so api docs are generated automatically!

Reply:
Moved to https://github.com/openrelik/openrelik-common/pull/1